recipe/common.php

```diff
@@ -71,16 +71,10 @@
 /**
  * Release
  */
-task('deploy:release', function () {
-    $release = date('YmdHis');
-
+$release = date('YmdHis');
+task('deploy:release', function () use ($release) {
     $releasePath = "{deploy_path}/releases/$release";
 
-    $i = 0;
-    while (is_dir(env()->parse($releasePath)) && $i < 42) {
-        $releasePath .= '.' . ++$i;
-    }
-
```

Comment:
Yes, it's needed in tests. Tree time runs `deploy:release` and check all was correct. 
[42](https://simple.wikipedia.org/wiki/42_(answer))